example/ios/Podfile

```diff
@@ -16,6 +16,9 @@ end
 
 target 'ReactNativeTurboLogExample' do
   config = use_native_modules!
+  
+  pod 'CocoaLumberjack', :modular_headers => true
```

Comment:
I would expect this is not neded here.

Reply:
Not adding it causes an error on pod install, due to the `modular_headers`. Not sure what is the triggering factor here, but going this way fixes the issue.